source/manifest/manifest_parser.py

```diff
@@ -152,6 +152,10 @@ def get_accounts_in_ou(self, ou_id_to_account_map, ou_name_to_id_map,
         ou_ids_manifest = []
         # convert OU Name to OU IDs
         for ou_name in resource.deploy_to_ou:
+
+            if ou_name == 'Root':
+                accounts_in_ou.extend(accounts_in_all_ous)
+
```

Comment:
accounts_in_all_ous is not defined in this context.  I believe a better change would be to lines 255 and 256 (now 259 and 260).
```
        _ou_name_to_id_map = {"Root":root_id}
        _all_ou_ids = [root_id]
```